lib/check-tag.js

```diff
@@ -0,0 +1,51 @@
+var logger = require('./out');
+
+var log = logger({
+});
+
+function checkTag(app, mod, name) {
+  if (mod.tag && mod.tag !== '' && typeof mod.tag === 'string') {
+    mod.tag = mod.tag.split(',');
+  }
+  if (app.withTags && typeof app.withTags === 'string') {
+    app.withTags = app.withTags.split(',');
+  }
+  if (app.excludeTags && typeof app.excludeTags === 'string') {
```

Comment:
- In what situation are `app.withTags`, or `app.excludeTags` not undefined or of type `string`? `withTags` and `excludeTags` are defined as strings in the option parser. You could just drop this entirely and do this in the `checkMatches()` calls later (see later comment).